### ISSUE TITLE
Remove progress bar in the preview application page

### ISSUE THREAD

**Issue by glorialiou** [civiform/civiform#1921]
Remove progress bar in the preview application page in the applicant experience: https://docs.google.com/presentation/d/1goxiepPXPrW3wK8wUSNoCpcf7iXIohXfj96iTYiQxo0/edit#slide=id.gfa7bf315fa_0_741

**Comment by kristinyim** [comment]
Reconfirmed by Rochester UXR. Ideally would like to ensure the progress bar matches the number of steps completed but this is hard because it depends on how many enumerators a resident has.

**Comment by kristinyim** [comment]
Update: UXR found some research from survey monkey that states: The visual scale displayed alone–without page numbers or percent complete–is best" for completion rates. So for this issue, we just want to remove the number next to the bar.

This still isn't ideal because the bar may not move or may move backwards at times but that'll be a later fix.

[Source](https://www.surveymonkey.com/curiosity/progress-bars-good-bad-survey-survey-says/?ut_source=mp&ut_source2=mobile-friendly-surveys&ut_source3=inline)

**Comment by azizova-leyla** [comment]
This seems to be fixed? Current state is progress bar without any numbers as Kristin described. 
![image](https://user-images.githubusercontent.com/1741747/163011839-726c3703-1342-479c-b506-354673ff45dd.png)

**Comment by bion** [comment]
@glorialiou @kristinyim is this fixed?

**Comment by glorialiou** [comment]
No -- so the issue is that with enumerated questions, the progress bar doesn't take that into account. So the progress bar will make it seem like the applicant is almost done and doesn't account for the fact that the applicant will have to do it all over again for their spouse, dependents, etc.

The quick-fix solution would be to just remove the progress bar altogether because it's currently confusing (as raised by Rochester and confirmed by UXR). Another solution would be to have the progress bar be able to account for enumerated questions, but not sure it's worth the headache to go that route.

**Comment by th174** [comment]
> So the progress bar will make it seem like the applicant is almost done and doesn't account for the fact that the applicant will have to do it all over again for their spouse, dependents, etc.

I agree with this, but I don't think removing the progress bar altogether is an improvement at all in user experience. I think progress bars are important for managing user expectations. Progress bars give users a sense of progress and keep the end of the form in sight.

If I'm working on a form without any indication of progress that I've already completed, I pretty quickly start feeling anxious, and I start wondering whether the form is even finite.

I think a better solution to the confusion would be to allow admins to specify an expected completion time field for each program or screen. We want users to know how much work to expect and how much time to reserve in their day before starting the form.

**Comment by bion** [comment]
> I think a better solution to the confusion would be to allow admins to specify an expected completion time field for each program or screen. 

This can vary quite a bit depending on a number of factors. E.g. for the vaccine eligibility an applicant skips almost the entire questionnaire if in an early answer they indicate employment in a high-risk field, so the time to complete ranges from 1m to 30m.

**Comment by rockycodes** [comment]
Closing as not planned since we will launch North Star soon.
